pkg/querier/stats/stats.proto

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// Provenance-includes-location: https://github.com/cortexproject/cortex/blob/master/pkg/querier/stats/stats.proto
+// Provenance-includes-license: Apache-2.0
+// Provenance-includes-copyright: The Cortex Authors.
+
+syntax = "proto3";
+
+package stats;
+
+option go_package = "stats";
+
+message Stats {
+  // The sum of all wall time spent in the querier to execute the query.
+  int64 wall_time = 1;
```

Comment:
Maybe wall_time_seconds?

Reply:
Those are from mimir, and not fully sure they will stay as it. I kept them for simplicity.